fairscale/experimental/nn/ampnet_pipe/ampnet.py

```diff
@@ -160,7 +160,7 @@ def event_loop_head_across_minibatches(
                 reqd_input = transform_logger_object.transform_input(cur_batch).to(self.input_device)
                 batch = Batch(reqd_input, count)
                 if self.weight_prediction:
-                    optimizer.update_weight_using_future_predictions(cur_rank, N, forward=True)  # type: ignore
+                    optimizer.update_weight_using_future_predictions(cur_rank, N, forward=True)
```

Comment:
these are some side effects due to stud change for mypy for the base optimizer class.